Woof/Woof/Helper/AppButton/AppButton.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+/// Represents name of buttons which uses in application.
```

Comment:
nit:
```suggestion
/// Represents name of button which is used in the application.
```

Reply:
Fixed with f4f7a7c

Reply:
As far as I know, it's common agreement to use singular nouns in the documentation.

Reply:
Fixed with 03725e9